go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/alecthomas/units v0.0.0-20190910110746-680d30ca3117 // indirect
 	github.com/andybalholm/cascadia v1.0.0 // indirect
 	github.com/aulanov/go.dbus v0.0.0-20150729231527-25c3068a42a0 // indirect
-	github.com/aws/aws-sdk-go v1.14.8
+	github.com/aws/aws-sdk-go v1.23.15
```

Comment:
I had to upgrade the SDK in order to get this to work. However, I did not upgrade to the latest version, only to a version I knew to work. If desired I'll bump to `v1.25.10` which [as of yesterday is the latest release](https://github.com/aws/aws-sdk-go/releases).

Reply:
For reference, `v1.14.8` was released in June of 2018 and `v1.23.15` was released last month.